pkg/contexts/ocm/blobhandler/handlers/generic/mvn/registration.go

```diff
@@ -0,0 +1,74 @@
+package mvn
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/open-component-model/ocm/pkg/contexts/ocm/cpi"
+	"github.com/open-component-model/ocm/pkg/contexts/ocm/resourcetypes"
+	"github.com/open-component-model/ocm/pkg/errors"
+	"github.com/open-component-model/ocm/pkg/mime"
+	"github.com/open-component-model/ocm/pkg/registrations"
+)
+
+type Config struct {
+	Url string `json:"url"`
+}
+
+type rawConfig Config
+
+func (c *Config) UnmarshalJSON(data []byte) error {
```

Comment:
Why are you unmarshaling twice the same thing into the same thing? 🤔

Reply:
see the tests:
`cfg := Must(registrations.DecodeConfig[mvn.Config]("test"))` and 
`cfg := Must(registrations.DecodeConfig[mvn.Config]('{"Url":"test"}'))` 

you can either provide directly the url or a complete json struct